### DISCUSSION TITLE
Easy upload to Home Assistant From Xdrip+

### DISCUSSION THREAD

**Discussion by 135576** [NightscoutFoundation/xDrip#2766] [Ideas]
I know we can go thru all the setup with nightscout and then add nightscout integration  into Home Assistant.   This is a lot of work for the average Joe. I use Xdrip only.

Would it be possible for a simple share setup to Home assistant or have all the nightscout setup automatically done for you in the app background?

**Comment by Navid200** [comment]
So, you don't have Nightscout now?
What happens to all your readings if your phone breaks or you lose it?

Have you seen these instructions for setting up Nightscout?
https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html
I'm just curious when you say it is a lot of work, do you mean that?

**Comment by 135576** [comment]
I ment that i don't use Dexcom App.  I upgraded phone to S23 ..... and Dexcom app doesn't work with it.  I've been uploading to Tidepool but it doesn't seem to log all the multiple insulin type injections correctly.  (Basil)

I hadn't seen the link you sent and i will look into it. Thank you.

**Comment by marciogranzotto** [comment]
@135576 you can just use the Nightscout Add-On and the Home Assistant Nighscout integration
https://nightscout.github.io/nightscout/advanced/#home-assistant-add-on-for-nightscout

**Comment by williknow** [comment]
I also would like their feature very much. I do not use nightscout. Hopefully we will find some more users interested in it.

**Comment by Navid200** [comment]
xDrip currently has more than 80 open issues.  Not all of those are feature requests.  Some are broken features that either no one knows how to fix or no one wants to fix!

This, having broken features we know but are not fixing, does not provide a good image for xDrip.
I would like us to fix such issues.

In my opinion, xDrip should do only a few things and do them really well instead of doing too many things really poorly.
In my opinion, we should never add a new feature unless it is absolutely necessary for that reason.
I don't want one more feature that breaks after Android 16 comes out and no one wants to fix.

Considering there is already a way to accomplish what this request is about, I hope that people choose to use the existing method.

If you try setting up Nightscout, using our methodology in Google Cloud, and run into trouble, I will be happy to walk you through the steps to complete it.
